samples/tab-navbar-menu/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-navbar-menu-ts",
+    "source": "officeDev",
+    "title": "Starter solution: Personal tab with Navbar Menu in TypeScript",
+    "shortDescription": "Add multiple actions to the upper right NavBar and build an overflow menu for extra actions in an app",
```

Comment:
Add multiple actions to the upper right in Nav-Bar and build an overflow menu for extra actions in an app.

Reply:
I have updated the changes.